### DISCUSSION TITLE
My current test build!  Small OLED Screen showing IP Address

### DISCUSSION THREAD

**Discussion by icsy7867** [tiny-pilot/tinypilot#1226] [Ideas]
Still a work in progress, I still need to mount the screen.  But the small SPI OLED screen is stupidly easy to configure
https://www.the-diy-life.com/connect-and-program-an-oled-stats-display-for-your-raspberry-pi/

<img width="317" alt="image" src="https://user-images.githubusercontent.com/3788162/206465041-d035e81b-182c-4ddf-8a9f-40d22dae537f.png">

For my organization, we have many different floors, subnets, vlans, etc... so there is no telling what IP address this thing is going to get.  So the small OLED that displays the IP address is super helpful.

My original idea was a small DB and web service that the pi would phone home to with the updated IP address, so someone could snag it from a web browser.  This would also be easy to do.  But I like the small 0.96" OLED.

**Comment by mtlynch** [comment]
Nice! Thanks for sharing! This is something we're interested in adding in a future version of the TinyPilot Voyager.
